backend/core/src/auth/services/user.service.spec.ts

```diff
@@ -76,6 +76,7 @@ describe("UserService", () => {
       const user: UserCreateDto = {
         email: "abc@xyz.com",
         emailConfirmation: "abc@xyz.com",
+        emailSubscription: false,
```

Comment:
It might be easier to make this field optional, basically just stick a "?" at the end of `emailSubscription` [right here](https://github.com/CityOfDetroit/bloom/blob/main/backend/core/src/auth/dto/user-create.dto.ts#L6). That way, future callers of createUser won't need to specify that field, and you could remove all these changes.

Reply:
makes sense, will do!